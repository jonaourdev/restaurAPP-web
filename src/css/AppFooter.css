.footerCSS {
  background-color: rgb(255, 184, 0);
}
