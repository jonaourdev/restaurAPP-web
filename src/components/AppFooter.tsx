import {Container} from "react-bootstrap";
import "../css/AppFooter.css";

function AppFooter() {
  return (
    <>
      <footer className=" py-3 mt-auto footerCSS">
        <Container className="text-center small text-dark">
          Â© 2025 - Excelsior - Todos los derechos reservados
        </Container>
      </footer>
    </>
  );
}

export default AppFooter;
