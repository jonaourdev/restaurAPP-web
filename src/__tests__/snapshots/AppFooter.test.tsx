import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';

// 1. Importa el componente que vas a probar
// ¡IMPORTANTE: Ajusta esta ruta para que apunte a tu componente!
import AppFooter from '../../components/AppFooter'; 

// --- Inicio de las Pruebas ---

describe('<AppFooter />', () => {

  it('debería renderizarse correctamente y coincidir con el snapshot', () => {
    // 1. Renderizamos el componente.
    // Usamos 'container' que nos da el <div> raíz donde se renderizó todo.
    const { container } = render(<AppFooter />);
    
    // 2. Le decimos a Vitest que compare el HTML del 'container'
    // con la "foto" (snapshot) que tiene guardada.
    expect(container).toMatchSnapshot();
  });

});