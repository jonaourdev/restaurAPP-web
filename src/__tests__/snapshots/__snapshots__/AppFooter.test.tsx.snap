// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<AppFooter /> > debería renderizarse correctamente y coincidir con el snapshot 1`] = `
<div>
  <footer
    class=" py-3 mt-auto footerCSS"
  >
    <div
      class="text-center small text-dark container"
    >
      © 2025 - Excelsior - Todos los derechos reservados
    </div>
  </footer>
</div>
`;
